CoC.txt

```diff
@@ -27,9 +27,40 @@ In addition, we take violations of this code outside these spaces into account
 when determining a person's ability to participate within them.
 
 ================================================================================
-Attribution
+Summary
 ================================================================================
 
-Parts of this code of conduct are derived from:
+The SecureDrop community should be a place where people feel safe and welcome.
+They should enjoy participating in discussions and contributing. To these ends,
+members of the community should:
 
+    Be friendly and patient
```

Comment:
I think `Assume good faith` is also an important one.

Reply:
Does the commit I just push adequately resolve that?